src/predictions/profiles_mlcorelib/py_native/attribution_report.py

```diff
@@ -613,21 +613,19 @@ def _create_user_journey_cte(
             counter += 1
         return journey_query, set_jouney_ref
 
-    def try_deref(self, this: WhtMaterial, model_ref: str):
-        try:
-            this.de_ref(model_ref)
-        except Exception:
-            # Skipping model_ref as it does not exist
-            return
-        self.inputs.add(model_ref)
-
     def _define_input_dependency(
         self, this: WhtMaterial, user_journeys, campaign_details, campaign_vars
     ):
+        entity_column_mapping = {
+            ENTITY_KEY: self.entity_id_column_name,
+            self.campaign_entity: self.campaign_id_column_name,
+        }
+
         self.inputs.add(f"{self.conversion_entity}/all/var_table")
         for obj in user_journeys:
             tbl = obj["from"]
-            column = this.de_ref(tbl).model.time_filtering_column()
+            de_ref_tbl = this.de_ref(tbl)
```

Comment:
Nit: can have a better name
```suggestion
            touchpoint_mat = this.de_ref(tbl)
```